config/api.yml

```diff
@@ -734,8 +734,11 @@
       :post:
       - :name: run
         :identifier: miq_report_run
+      - :name: schedule
+        :identifier: miq_report_run
```

Comment:
not sure about this role identifier, maybe miq_report_schedule_add ?    @dclarizio ?